common/app/com/commercetools/sunrise/play/SunriseHttpFilters.java

```diff
@@ -1,5 +1,7 @@
-package com.commercetools.sunrise.common.basicauth;
+package com.commercetools.sunrise.play;
```

Comment:
@schleichardt I've created the `play` package, where I pretend to put very Play-related things. For example, in this case it's the `Filters` or the `ApplicationLoader` that are used to alter the behaviour of Play. Open to suggestions.